<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Player</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : rgb(52,51,64);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="mediaBrowserWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Media Browser</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchLineEdit">
          <property name="placeholderText">
           <string>Search media files...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="mediaTreeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="playlistWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Playlists</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="playlistComboBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="newPlaylistButton">
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deletePlaylistButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="playlistView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="addToPlaylistButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeFromPlaylistButton">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playButton">
            <property name="text">
             <string>Play</string>
            </property>
            <property name="icon">
             <iconset theme="media-playback-start"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionBrowseFolder"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Audio"/>
    <addaction name="actionOpen_Video"/>
   </widget>
   <widget class="QMenu" name="menuPlaylist">
    <property name="title">
     <string>Playlist</string>
    </property>
    <addaction name="actionNewPlaylist"/>
    <addaction name="actionDeletePlaylist"/>
    <addaction name="separator"/>
    <addaction name="actionSavePlaylist"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlaylist"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Audio">
   <property name="text">
    <string>Open Audio</string>
   </property>
  </action>
  <action name="actionOpen_Video">
   <property name="text">
    <string>Open Video</string>
   </property>
  </action>
  <action name="actionBrowseFolder">
   <property name="text">
    <string>Browse Folder</string>
   </property>
  </action>
  <action name="actionNewPlaylist">
   <property name="text">
    <string>New Playlist</string>
   </property>
  </action>
  <action name="actionDeletePlaylist">
   <property name="text">
    <string>Delete Playlist</string>
   </property>
  </action>
  <action name="actionSavePlaylist">
   <property name="text">
    <string>Save Playlist</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
